models/src/main/java/si/fri/tabletop/places/Place.java

```diff
@@ -40,8 +46,8 @@ public String getTitle() {
         return title;
     }
 
-    public void setTitle(String title) {
-        this.title = title;
+    public void setTitle(String name) {
```

Comment:
Minor, but would it not make more sense to use `title`instead of `name` as function argument?

Reply:
True story bro! I couldn't decide between name/title, so I changed back and forth and forgot to change all.